README.md

```diff
@@ -1,3 +1,5 @@
+> # This codelab is deprecated and it will be removed soon.
```

Comment:
I used a red box:

> [!CAUTION]
> This codelab is deprecated and it will be removed soon.

which is so awesome.

    > [!CAUTION]"""
    > This codelab is deprecated and it will be removed soon."""